### ISSUE TITLE
Trained model not working

### ISSUE THREAD

**Issue by Jay4BioPZ** [jono-m/OrganoID#7]
Hi there,
I am using your algorithm to perform segmentation on my dataset. As a proof of concept, I retrained the model using the MouseOrganoid dataset you offered. The training command I used is:

`
python OrganoID.py train --lite C:\Users\ychiang\Desktop\Github\OrganoidSecretion.OrganoID\img\example_dataset\MouseOrganoids C:\Users\ychiang\Desktop\Github\OrganoidSecretion.OrganoID\OrganoID-master\Example_model example_model
`

A warning message popped out at the end of the training:

`
13/13 [==============================] - 37s 3s/step - loss: -460.7747 - val_loss: -276.9594
Epoch 11: early stopping
WARNING:absl:Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 24). These functions will not be directly callable after loading.
2023-07-03 14:20:47.680599: W tensorflow/compiler/mlir/lite/python/tf_tfl_flatbuffer_helpers.cc:364] Ignored output_format.
2023-07-03 14:20:47.680734: W tensorflow/compiler/mlir/lite/python/tf_tfl_flatbuffer_helpers.cc:367] Ignored drop_control_dependency.
`

Furthermore, as I run the segmentation with the testing set, nothing was segmented. Could you please help me out with this?
